lib/popup.html

```diff
@@ -11,7 +11,9 @@
       <div></div>
     </header>
     <div class="non-measure-display">
-      <h3>This website is not tracked!</h3>
+      <h3>It looks like this website is not on your list.</h3>
+      <h3>If it should, you can add it now.</h3>
```

Comment:
Can you reduce the space between these two `h3` elements and increase little bit the space between the text and the button?